scripts/go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/aquasecurity/trivy-checks/bundle
```

Comment:
why does this file (and also the go.sum) have git addition marks? (The plus sign appended to each of the lines).

Reply:
it's a new file

Reply:
doh 🤦🏼 you're right!